demo/image_demo.py

```diff
@@ -1,98 +1,92 @@
 # Copyright (c) OpenMMLab. All rights reserved.
-import asyncio
+import warnings
 from argparse import ArgumentParser
 
-import mmcv
+from mmengine.logging import print_log
 
-from mmdet.apis import (async_inference_detector, inference_detector,
-                        init_detector)
-from mmdet.registry import VISUALIZERS
-from mmdet.utils import register_all_modules
+from mmdet.apis import MMDetInferencer
 
 
 def parse_args():
     parser = ArgumentParser()
-    parser.add_argument('img', help='Image file')
-    parser.add_argument('config', help='Config file')
-    parser.add_argument('checkpoint', help='Checkpoint file')
-    parser.add_argument('--out-file', default=None, help='Path to output file')
+    parser.add_argument(
+        'inputs', type=str, help='Input image file or folder path.')
+    parser.add_argument('model', help='Config file')
```

Comment:
Path to the config file or the model name defined in metafile.

Reply:
resolved